### ISSUE TITLE
[Issue] When the order of Parameters in Visject is set, the sort of these Parameters in other windows is always alphabetical.

### ISSUE THREAD

**Issue by ElevenArt** [FlaxEngine/FlaxEngine#2825]
When the order of Parameters is changed in Visject Editor, in these panels it sorts the Parameters alphabetically:
- Actor Script Properties
- Material Instance
- Particle System

For example
Material:
![Visject-Mat](https://github.com/user-attachments/assets/0214123d-ff70-45b4-b196-41b164fc783a)

Material Instance:
![Visject-Mat-instance](https://github.com/user-attachments/assets/11372fa9-851c-445a-8cb2-e6d921e765b3)

Flax version:
1.8.2 Master

**Comment by ElevenArt** [comment]
A possible solution for this issue is described in [discord](https://discord.com/channels/437989205315158016/1239886455011020800/1255636348866396170)

**Comment by mafiesto4** [comment]
That's because we don't support editing parameter attributes in the material instances by design. Only values van be overridden.

**Comment by ElevenArt** [comment]
> That's because we don't support editing parameter attributes in the material instances by design. Only values van be overridden.

I don't know if I described it correctly. The point is that when I change the order of parameters in Material, the order of Parameters in Material Instance is alphabetical, regardless of the order of parameters I specified in Material. That's what this issue is about. 

It is the same with Visual Script, where the order of parameters in Actor Properties is alphabetical and not according to the order in Visual Script. The same is when I specify the order of parameters in Particle Emitter, so in Particle System the order of parameters is alphabetical and not according to the order specified in Particle Emitter.

Is there any way to fix this?
